audb/core/dependencies.py

```diff
@@ -62,7 +99,7 @@ def archives(self) -> typing.List[str]:
 
         """
         archives = [self.archive(file) for file in self.files]
-        return list(set(archives))
+        return sorted(list(set(archives)))
```

Comment:
Why do you think it's important to sort here?

Reply:
Otherwise the docstring test will fail:

```python
>>> deps.archives[:2]
['005d2b91-5317-0c80-d602-6d55f0323f8c', '014f82d8-3491-fd00-7397-c3b2ac3b2875']
```
and I find it better if the behavior is deterministic.